### ISSUE TITLE
Issues placing Shulker Boxes

### ISSUE THREAD

**Issue by minoneer** [PaperMC/Paper#483]
Placing a ShulkerBox on the side of another ShulkerBox caused this error:
http://hastebin.com/biyabaqefa.vbs

Trying to reproduce this by placing many boxes against each other caused a different exception:
http://hastebin.com/tutucoxozu.md

not sure if they are related, so far no luck reproducing, but something is odd about them.

Paper version git-Paper-918 (MC: 1.11) (Implementing API version 1.11-R0.1-SNAPSHOT)

**Comment by zachbr** [comment]
19:41   ~Z750   minoneer: I cant reproduce that :(
19:41   minoneer    me neither
19:42   minoneer    it happened only once - with one other (maybe related) error when trying to reproduce

Please let us know if anyone else sees this behavior.

**Comment by BillyGalbreath** [comment]
https://bugs.mojang.com/browse/MC-109593

Happened to me too once. Not sure what I was doing that caused it. All I remember is I was playing with shulker boxes in a cave, and was unable to reproduce. However I did find this while googling.

**Comment by aikar** [comment]
The first stack wasnt a 'problem' as I described on the linked MC Bug ticket. It was simply a debug message. Grum fixed it in 1.11.2 to not send that message anymore.

The 2nd linked stack is a completely different error that occurs with incorrect chunk data, when something goes wrong and gets inconsistent. Solving all chunk corruption issues is a can of worms no one can do.

So ultimately there wasn't an "issue", was just a debug message that has been removed in 1.11.2.
